resources/assets/coffee/store.coffee

```diff
@@ -77,6 +77,17 @@ $(document).on 'input', '#username.form-control', ->
     $status.html "Checking availability of #{requestedUsername}..."
     debouncedCheckUsernameValidity()
 
+#checkout checks
+checkCheckoutConfirmations = ->
+  $checkboxes = $('.checkout-confirmation-step')
+  $checkboxesChecked = $('.checkout-confirmation-step:checked')
+  $('#checkout-with-paypal').toggleClass('disabled', $checkboxesChecked.length < $checkboxes.length)
```

Comment:
Using `.js-checkout-confirmation--button` for this and `.js-checkout-confirmation--step` for the checkboxes will perhaps be easier to understand? (or some better class names)